### DISCUSSION TITLE
How to embed iframe form in my site

### DISCUSSION THREAD

**Discussion by behoppe** [11ty/eleventy#3354] [Q&A] [answered]
I would like to implement a "Join Us" page on my eleventy site with a short survey. I tried something like the following, but the iframe gets rendered as empty. How can I embed a survey (e.g. Qualtrics, Google forms, etc.) into my eleventy site? Thank you for any advice.
```
---
layout: layouts/page.njk
title: Join us
eleventyNavigation:
  key: Join us
  order: 1
---

The OpenCilk project welcomes your expertise and enthusiasm. Please fill out the form below to get started. Thank you for your interest!

<iframe src="https://survey.az1.qualtrics.com/jfe/form/SV_8jGNNwhkEvx6CIC" height="800px" width="600px"></iframe>
```
PS: I have successfully embedded HubSpot forms by following [these instructions](https://knowledge.hubspot.com/forms/how-can-i-share-a-hubspot-form-if-im-using-an-external-site), which use Javascript. So, my question might point in two different directions: (1) how can I get another form provider to generate javascript like HubSpot, or (2) how can I get eleventy to embed qualtrics or Google forms in the same way it can embed "everything else" (https://www.npmjs.com/package/eleventy-plugin-embed-everything)?

**Comment by zachleat** [comment, marked as answer]
I’m guessing you have an Ad blocker enabled on your browser. If that isn’t true, you might want to check to see if your site has a CSP https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP (though this is unlikely if it’s just your local machine)

Note also that the iframe `height` and `width` attributes should be numbers: `height="800" width="600"`

Otherwise, Eleventy shouldn’t be doing anything to stop this code from working!

**Reply by behoppe** [reply]
@zachleat thank you. Yes -- after I disabled the ad blocker on my browser, my Eleventy iframe worked fine. So, my original problem is fixed, but now I'm wondering how to deploy a short survey on my site that doesn't get killed by an ad blocker. How would you approach this? (I have basic skills with HTML, and I'm resourceful about interfacing things, but I'm a javascript rookie.) Thank you!!
